src/App.tsx

```diff
@@ -1,14 +1,71 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { SelectStatusTodos } from './types/SelectStatusTodos';
+
+interface FilterTodos {
+  searchByTitle: string;
+  selectStatusTodos: SelectStatusTodos;
+}
+
+const prepareTodos = (todos: Todo[], filter: FilterTodos): Todo[] => {
+  let preparedTodos = [...todos];
+
+  if (filter.searchByTitle !== '') {
```

Comment:
```suggestion
  if (filter.searchByTitle) {
```
You don't need to compare with an empty string, just check if a value exists
https://developer.mozilla.org/en-US/docs/Glossary/Truthy